yarn.lock

```diff
@@ -1127,9 +1127,9 @@
     glob "^7.2.0"
 
 "@hypothesis/frontend-shared@^5.7.0":
-  version "5.9.0"
-  resolved "https://registry.yarnpkg.com/@hypothesis/frontend-shared/-/frontend-shared-5.9.0.tgz#1ccd6007e237c90fee77689417bfd19a0fde0498"
-  integrity sha512-FyeTvDLJ55Ma676f3pJoy1Ybst++eiNnDAuXgIH0tlYBE+OTg2paQBnC7evb19E9O1m0m9zpTZV0TKjjQTF7SA==
+  version "5.9.1"
```

Comment:
Upgrading to latest version of the pattern library, which makes sure the default value for `type` attribute in buttons is `button`.